app/api/services/evidence.py

```diff
@@ -82,6 +82,32 @@ def get_previous_submitted_evidence_for_supplier_and_domain(self, evidence_id, d
         evidence = query.first()
         return evidence
 
+    def get_all_evidence(self, supplier_code=None):
+        query = (
+            db.session.query(
+                Evidence.id.label('id'),
+                Evidence.status,
+                Evidence.created_at,
+                Evidence.submitted_at,
+                Evidence.approved_at,
+                Evidence.rejected_at,
+                Evidence.data.label('data'),
+                Evidence.data['maxDailyRate'].astext.label('maxDailyRate'),
+                Supplier.name.label('supplier_name'), Supplier.code.label('supplier_code'),
+                Brief.id.label('brief_id'), Brief.closed_at.label('brief_closed_at'),
+                Brief.data['title'].astext.label('brief_title'),
+                Domain.name.label('domain_name'), Domain.id.label('domain_id'),
+                Domain.price_maximum.label('domain_price_maximum')
+            )
+            .join(Domain, and_(Evidence.domain_id == Domain.id))
+            .join(Supplier, and_(Evidence.supplier_code == Supplier.code))
+            .outerjoin(Brief, and_(Evidence.brief_id == Brief.id))
```

Comment:
is the and_ really required here?

```suggestion
            .outerjoin(Brief, Evidence.brief_id == Brief.id)
```